man/cpg_model_initialize.3.in

```diff
@@ -204,7 +204,9 @@ struct cpg_ring_id {
 .PP
 where
 .I nodeid
-is if of node of current Totem leader and seq is increasing number.
+is current Totem leader and
```

Comment:
"is the current Totem leader and"

Reply:
Hehe, thanks. These the/a/an is really my Achilles' heel. Fixed now.